<launch>
  <node name="semantic_camera_node" pkg="semantic_camera" type="semantic_camera_node" output="screen" />
  <node name="traffic_light_controller" pkg="semantic_camera" type="traffic_light_controller" output="screen" />
</launch>
